pkg/sensors/tracing/policyhandler_windows.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package tracing
+
+import (
+	"errors"
+
+	"github.com/cilium/tetragon/pkg/policyfilter"
+	"github.com/cilium/tetragon/pkg/sensors"
+	"github.com/cilium/tetragon/pkg/tracingpolicy"
+)
+
+type policyHandler struct{}
+
+func init() {
+	sensors.RegisterPolicyHandlerAtInit("tracing", policyHandler{})
+}
+
+func (h policyHandler) PolicyHandler(
+	policy tracingpolicy.TracingPolicy,
+	policyID policyfilter.PolicyID,
+) (sensors.SensorIface, error) {
+
+	policyName := policy.TpName()
+	spec := policy.TpSpec()
+
+	namespace := ""
+	if tpn, ok := policy.(tracingpolicy.TracingPolicyNamespaced); ok {
+		namespace = tpn.TpNamespace()
+	}
+
+	sections := 0
+	if len(spec.KProbes) > 0 {
+		sections++
+	}
+	if len(spec.Tracepoints) > 0 {
+		sections++
+	}
+	if len(spec.LsmHooks) > 0 {
+		sections++
+	}
+	if sections > 1 {
+		return nil, errors.New("tracing policies with multiple sections of kprobes, tracepoints, or lsm hooks are currently not supported " + policyName + namespace)
+	}
+
+	//ToDo: Create Windows specific sensor here
+	return nil, nil
```

Comment:
As far as I can tell, the windows specific sensor is not created in this PR.
So I think the proper behavior is to return an error if `sections > 0` since none of these are currently supported.